conf/application.conf

```diff
@@ -70,25 +64,9 @@ play.filters.headers.contentSecurityPolicy= "default-src 'unsafe-inline' 'self'
 # !!!WARNING!!! DO NOT CHANGE THIS ROUTER
 play.http.router=prod.Routes
 
-# Metrics plugin settings - graphite reporting is configured on a per env basis
-metrics {
-    name = ${appName}
-    rateUnit = SECONDS
-    durationUnit = SECONDS
-    showSamples = true
-    jvm = true
-    enabled = true
-}
-
-
 # Microservice specific config
 
 auditing {
     enabled=false
-    consumer {
-        baseUri {
-            host = localhost
-            port = 8100
-        }
     }
```

Comment:
Need to remove this closing brace, as it causes an `application.conf: 72:` unbalanced close brace error if you spin the service up locally and try to view the guide.